backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -36,7 +36,7 @@ public static IQueryable<Entity.Views.Property> GenerateQuery(this PimsContext c
             // Users may only view sensitive properties if they have the `sensitive-view` claim and belong to the owning agency.
             var query = context.Properties
                 .AsNoTracking()
-                .Where(p => p.ClassificationId != 4); // Disposed properties are not visible.
+                .Where(p => p.ClassificationId != 4 && p.ClassificationId != 5 && p.ClassificationId != 6); // Disposed and subdivided/demolished properties are not visible.
```

Comment:
Sorry, I think something got lost in translation here. If we are going to be using ids in a conditional like this, I'd rather we use an enum like what you were doing earlier to prevent magic numbers and make the code more readable. It isn't the best way to handle codified values (thus my quick and dirty comment from before), but it is at least readable.

The other option is to use an enum corresponding to the classification code values. For that you'd need to modify the view to return the code, currently it returns the classificationId and the classification name.

Reply:
Ahhh oops haha I did misread the comments, adding back now.